scripts/upload-source-maps.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import { execSync } from 'child_process'
+
+const main = () => {
+	if (process.env.VERCEL_ENV === 'development') {
+		return
+	}
+
+	const LATEST_SHA = process.env.VERCEL_GIT_COMMIT_SHA
+	const PATH_PREFIX = 'https://developer.hashicorp.com/_next/static/'
+	const SERVICE = 'developer.hashicorp.com'
+
+	const DATADOG_API_KEY = process.env.DD_API_KEY
+
+	try {
+		const status = execSync(
+			`DATADOG_API_KEY=${DATADOG_API_KEY} npx @datadog/datadog-ci sourcemaps upload .next/static/ --service=${SERVICE} --release-version=${LATEST_SHA} --minified-path-prefix=${PATH_PREFIX}`
+		)
+
+		status
+			.toString()
+			.split('\n')
+			.forEach((line) => {
+				console.log(line)
+			})
```

Comment:
As an alternative to storing `status` in a `const` and then logging it out _after_ the command is done, I think we could use the [`stdio` option](https://nodejs.org/api/child_process.html#child_processexecsynccommand-options) here to log output from the `execSync` command as it happens:

```suggestion
        { stdio: 'inherit' }
		)
```